include/fluent-bit/flb_mem.h

```diff
@@ -55,6 +56,8 @@ static inline ALLOCSZ_ATTR(1)
 void *flb_malloc(const size_t size) {
     void *aux;
 
+    usleep(2000);
```

Comment:
This is just to simulate the problem. This commit is not part of the cbtimer resolution